cmd/routing-api/main.go

```diff
@@ -87,6 +89,7 @@ func main() {
 	}
 
 	prefix := "routing_api"
+	// lint :ignore SA1019
```

Comment:
This lint ignore is here because this function is considered deprecated in the latest version. If needed, I can write a separate story to look into replacing this function.